.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main,airflow-version-tests]
```

```suggestion
    branches: [main]
```
Remove before merging.